angular/projects/spark-angular/src/lib/components/inputs/sprk-radio-group/sprk-radio-group.component.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Component,
+  AfterContentInit,
+  Input,
+  ContentChild,
+  ContentChildren,
+  QueryList,
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+import { SprkRadioItemComponent } from '../sprk-radio-item/sprk-radio-item.component';
+import { SprkFieldErrorDirective } from '../../../directives/inputs/sprk-field-error/sprk-field-error.directive';
+import { SprkLabelDirective } from '../../../directives/inputs/sprk-label/sprk-label.directive';
+
+@Component({
+  selector: 'sprk-radio-group',
+  template: `
+    <div [ngClass]="getClasses()" [attr.data-id]="idString">
+      <ng-content select="[beforeSprkRadios]"></ng-content>
+      <fieldset *ngIf="label; else sprkRadios" class="sprk-b-Fieldset">
+        <legend class="sprk-b-Legend">
+          <ng-content select="[sprkLabel]"></ng-content>
```

Comment:
the legend element shouldnt have a label element inside it, just text

Reply:
Fixed!